spec/models/exercise_spec.rb

```diff
@@ -1,10 +1,42 @@
 require 'rails_helper'
 
 RSpec.describe Exercise, type: :model do
+  let(:current_user) { FactoryBot.create(:user) }
+  let(:exercise) { FactoryBot.create(:exercise, user: current_user) }
+
+  describe 'Basic setup methods' do
+    it 'has a valid factory' do
+      expect(exercise).to be_valid
+    end
+
+    it 'persists a valid model object when given strings for title, general description, user id and a global boolean' do
+      exercise = Exercise.new(
+        title: 'Senecas Dance with Dragons',
+        general_description: 'An amazing exercise to really get your pants off.',
+        user: current_user
+      )
```

Comment:
This doesn't test anything at all. Apart from that, I don't think you'll need to test persistence, as you'll start testing core functionality of Rails. 

Testing validity here should be enough:
`it 'is valid given strings for title...`
and the later:
`expect(exercise).to be_valid`

Reply:
Yes, pardon, good point. I missed the actual exercise here! 

I just added that.

I also updated the language of the test to start with your recommendation.